src/ms.tex

```diff
@@ -205,52 +221,90 @@ \subsection{Contributor base} \label{sec:project-contributors}
 % Transition of needs: From new features, to community development, sustainability, ...
 % - pipeline between User, Participant, Code contributor, Maintainer, Coordinator.
 
+\subsection{Inclusion, Diversity, and Equity Programs} \label{sec:project-ide}
+
+\secauthor{Lía Corrales}
 
 
 \subsection{Funding} \label{sec:project-funding}
 
-New funding sources.
-% Funding received:
-% - Moore
-% - NASA
+\secauthor{Aarya Patil}
+
+Summarize funding sources (Moore, NASA) and amounts and what this has been used
+for.
 
 
 \section{Supporting the Ecosystem of Astronomical Python Software}
 \label{sec:ecosystem}
 
 \subsection{Community-oriented infrastructure}
 
-Infrastructure packages that exist to support and help others maintain their package infrastructure.
+\secunfilled
+
+Infrastructure packages that exist to support and help others maintain their
+package infrastructure.
 
 \subsection{Affiliated packages}
 
-New packages and major updates!
+\secauthor{Matt Craig, Brett Morris}
```

Comment:
@mwcraig @bmorris3 : Please us a "Suggestion" to replace your name with `\secunfilled` if you are not able to contribute text to this section.

Reply:
I'm happy to work on this part